### DISCUSSION TITLE
me salen errores al seguir las instrucciones del modulo 0

### DISCUSSION THREAD

**Discussion by RupertoTacuche** [LaunchX-InnovaccionVirtual/CursoIntroPython#134] [Q&A] [answered]
Hola amigos, soy nuevo en python y visual studio code. He seguido las instrucciones al pie de la letra de lo que ha puesto para el modulo 0, Fhernanda, pero me salen estas cosas: 
![image](https://user-images.githubusercontent.com/86775311/153637742-04ddc43c-39a6-4399-8c82-b9dbc8d3bc8f.png)
y esto tambien:
![image](https://user-images.githubusercontent.com/86775311/153637924-73728a3b-577d-4f8a-982c-cdd60bc88c14.png)

A alguien le ha ocurrido lo mismo?
Me pueden ayudar?

**Comment by mxgmr** [comment]
Hola, intenta agregando un signo de admiración  al pip, de forma que sea !pip

**Reply by RupertoTacuche** [reply]
Gracias

**Comment by gersondlarosa** [comment]
a mi no me deja iniciar el pip me indica que no puede iniciar jupiter ya llevo dos días a si

**Comment by AlejandroMontoyaT** [comment]
podrian describir como hacen la instalacion. y el error que publicaste @RupertoTacuche al parecer es por que no se agrego el PATH podrias buscar en google. Agregar Python al path de Windows o si tu sistema operativo es uno diferente poner la version que es.

**Comment by MauroMontan** [comment]
No es realmente un problema, esto pasa cuando la carpeta de scripts ejecutables de python no se encuentran en el path, para agregarlos tienes que ir a las variables de entorno del sistema y agregar el path que te dice.
![image](https://user-images.githubusercontent.com/68311575/153641929-25dd7b2d-7b04-49b9-8962-51fa436ebd62.png)

**Comment by danielmares32** [comment, marked as answer]
### Para agregar al PATH
### Busca variables de entorno
![image](https://user-images.githubusercontent.com/53630621/153645203-fc4fc3d1-997a-4cda-9a81-2224098c3e3f.png)
### Luego seleccionas variables de entorno
![image](https://user-images.githubusercontent.com/53630621/153645249-8bc64402-d33a-4cd7-b724-88fe5ad1d5a4.png)
### Despues editas la que se llama PATH
![image](https://user-images.githubusercontent.com/53630621/153645297-65bb70f7-20f6-4369-a404-b45e8975ef22.png)
### Le das agregar una nueva
![image](https://user-images.githubusercontent.com/53630621/153645367-5c8a67e2-3bb6-4d5b-a080-59427d7c023a.png)
### Finalmente, ahí escribes el directorio donde lo tienes instalado

**Reply by RupertoTacuche** [reply]
Gracias, me orientaste mucho. Un abrazo
